frontend/javascripts/dashboard/advanced_dataset/dataset_table.js

```diff
@@ -230,6 +230,7 @@ class DatasetTable extends React.PureComponent<Props, State> {
         />
         <Column
           title="Data Layers"
+          key="dataLayers"
```

Comment:
I just noticed that react was complaining because the key prop was missing. I just added it here, as an own PR for this change would be overkill.